test/src/main/java/io/atomix/copycat/test/PerformanceTest.java

```diff
@@ -0,0 +1,527 @@
+/*
+ * Copyright 2017-present Open Networking Laboratory
```

Comment:
🤔 I wasn't paying attention to my copyright notice, but I think work I do for ON.Lab is under their copyright? I hate this legal crap...